mmdet3d/models/decode_heads/cylinder3d_head.py

```diff
@@ -19,61 +22,17 @@ class Cylinder3DHead(Base3DDecodeHead):
     `official code <https://https://github.com/xinge008/Cylinder3D>`_.
 
     Args:
-        channels (int): Channels after modules, before conv_seg.
-        num_classes (int): Number of classes.
-        dropout_ratio (float): Ratio of dropout layer. Defaults to 0.
-        conv_cfg (dict or :obj:`ConfigDict`): Config of conv layers.
-            Defaults to dict(type='Conv1d').
-        norm_cfg (dict or :obj:`ConfigDict`): Config of norm layers.
-            Defaults to dict(type='BN1d').
-        act_cfg (dict or :obj:`ConfigDict`): Config of activation layers.
-            Defaults to dict(type='ReLU').
-        loss_ce (dict or :obj:`ConfigDict`): Config of CrossEntropy loss.
-            Defaults to dict(
-                     type='mmdet.CrossEntropyLoss',
-                     use_sigmoid=False,
-                     class_weight=None,
-                     loss_weight=1.0).
-        loss_lovasz (dict or :obj:`ConfigDict`): Config of Lovasz loss.
-            Defaults to dict(type='LovaszLoss', loss_weight=1.0).
-        conv_seg_kernel_size (int): The kernel size used in conv_seg.
-            Defaults to 3.
-        ignore_index (int): The label index to be ignored. When using masked
-            BCE loss, ignore_index should be set to None. Defaults to 19.
-        init_cfg (dict or :obj:`ConfigDict` or list[dict or :obj:`ConfigDict`],
-            optional): Initialization config dict. Defaults to None.
```

Comment:
Why do need to remove these parameters in Cylinder3Dhead?

Reply:
These parameters are implemented in `Base3DDecodeHead` and we can simplify them with `**kwargs`.